clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -36,8 +36,13 @@ include "mlir/IR/SymbolInterfaces.td"
 // CIR Ops
 //===----------------------------------------------------------------------===//
 
+class LoweringInfo {
+  bit SimpleLowering = 0;
+  string LLVMOpName;
```

Comment:
Let's lowercase all member to be a closer match to MLIR, so this one would be `llvmOpName`;